app/Judite/Models/Student.php

```diff
@@ -140,7 +141,7 @@ public function unenroll(Course $course): bool
             throw new StudentIsNotEnrolledInCourseException($course);
         }
 
-        if (! $enrollment->isDeletable()) {
+        if (! ($enrollment->isDeletable())) {
```

Comment:
Why this?

Reply:
pretty sure linter was complaining but it does not now, reverting 🤦‍♂️  .